s3config.release.yaml.sample

```diff
@@ -70,6 +70,7 @@ S3_SERVER_CONFIG:                                       # Section for S3 Server
    S3_SERVER_MOTR_ETIMEDOUT_MAX_THRESHOLD: 5            # Number of ETIMEDOUT errors per monitoring window before s3server restart
    S3_SERVER_MOTR_ETIMEDOUT_WINDOW_SEC: 60              # Monitoring window for motr ETIMEDOUT errors in seconds
    S3_SERVER_ENABLE_ADDB_DUMP: true                     # If set to true, then addb dump will be collected
+   S3_MAX_EXTENDED_OBJECTS_IN_FAULT_MODE: 0		# Maximum number of allowed recovery by S3 when fault happens and Motr error out with specific error code. When 0, feature is disabled. The maximum value is 10.
```

Comment:
maximum we can reduce to 5. 10 is high

Reply:
This is configurable. we can always set to any value. I can set it to 5 on your request, but I don't know why only 5, why not 8 or 10 itself.

Reply:
In calls we said that we will be supporting one or twice failure, 10 is pretty big. Have it at lower value

Reply:
By default, value is 0 to make feature disabled. But, I've added in the comment of this property to set max to 5.